### DISCUSSION TITLE
Notebook to view/filter activities and laps

### DISCUSSION THREAD

**Discussion by Fvegini** [tcgoetz/GarminDB#205] [Show and tell]
I'm creating a notebook to help me to keep track of my current trainings "performance".
It may help other people... but I'm not sure how to share it properly.

It has a filter by lap distance and/or by lap pace, here is a screenshot example.

![Ipywidget](https://github.com/tcgoetz/GarminDB/assets/32243959/f5cee3cb-73f5-4c29-8128-683790bd88a8)

**Comment by tcgoetz** [comment]
That looks like a notebook a lot of pople could use. Do you want to create pull request against GarminDb's develop branch with a version that has english labels?

**Reply by Fvegini** [reply]
I tried to create this pull request... never did one before so not sure if I done it right!
